src/hssm/param.py

```diff
@@ -531,3 +654,14 @@ def _make_default_prior(bounds: tuple[float, float]) -> bmb.Prior:
         return bmb.Prior("TruncatedNormal", mu=lower, lower=lower, sigma=2.0)
     else:
         return bmb.Prior(name="Uniform", lower=lower, upper=upper)
+
+
+def merge_dicts(dict1: dict, dict2: dict) -> dict:
```

Comment:
strictly speaking this is not a merge right?

seems like if `key in merged` but instances are not dicts, you override the value of dict1 with whatever you find in dict2.

So maybe something like `override_dict` ?

Reply:
This actually raises a question: should the override be recursive? For example, if `1|participant_id` has a defined prior, with only `sigma` having a defined hyperprior, should we also add our default `mu`? It seems that if we don't then a `bambi` or `pymc` default will be applied

Reply:
I would say no, because we would in fact not cover the case in all generality even then. 
The logic would only apply if the user defined distribution `1|participant_id` matches the one for which we have hyper-priors available. 

If the user defines their own prior, they should define it all the way down I would say.